### PR TITLE
Update metering attributes

### PR DESCRIPTION
**Pull request by TedTolboom**
Validated to be working correctly on DP / Frient metering interface device

### CODE REVIEW

File: lib/clusters/metering.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const ATTRIBUTES = {
   currentMaxDemandDelivered: { id: 2, type: ZCLDataTypes.uint48 },
   currentMaxDemandReceived: { id: 3, type: ZCLDataTypes.uint48 },
   powerFactor: { id: 6, type: ZCLDataTypes.int8 },
+  meterStatus: { id: 512, type: ZCLDataTypes.map8('checkMeter', 'lowBattery', 'tamperDetect', 'powerFailure', 'powerQuality', 'leakDetect', 'leakDetect', 'serviceDisconnectOpen') },
```
Comment (RobinBol) [inline lib/clusters/metering.js:12]: I think you added `leakDetect` twice, can not find that in the spec.

**Review by RobinBol — CHANGES_REQUESTED** [review]

File: lib/clusters/metering.js (hunk 1 above)
Comment (TedTolboom) [reply lib/clusters/metering.js:12]: you're right should be 
`meterStatus: { id: 512, type: ZCLDataTypes.map8('checkMeter', 'lowBattery', 'tamperDetect', 'powerFailure', 'powerQuality', 'leakDetect', 'serviceDisconnectOpen', 'reserved') },`

**Comment by TedTolboom** [comment]
However, due to the dependency of the status attribute on the meter commodity type (see 10.4.2.2.3.1 Status Attribute), I do question if it makes sense to add it like this to the clusters definition. What do you think @RobinBol?

**Comment by RobinBol** [comment]
Hm, right, since it depends on the commodity type we can't define it like this. I think this should be parsed manually by the app. We can define it like `meterStatus: { id: 512, type: ZCLDataTypes.data8 }` or similar.

**Comment by RobinBol** [comment]
Closing this due to inactivity.

**Pull request closed without merging**